x/auth/client/cli/query.go

```diff
@@ -24,7 +25,7 @@ const (
 )
 
 // GetQueryCmd returns the transaction commands for this module
-func GetQueryCmd(cdc *codec.Codec) *cobra.Command {
+func GetQueryCmd(clientCtx client.Context) *cobra.Command {
```

Comment:
Just an FYI, this will all be undone once https://github.com/cosmos/cosmos-sdk/pull/6573 is merged. I'm not sure about the order of preference here, but just keep that in mind.

Essentially, command constructors will take no arguments anymore (at least not `client.Context`). They will instead use a `clientCtx := client.GetClientContextFromCmd(cmd)` call.

Reply:
Yes, I think we can wait until #6573 is merged.

Reply:
@alexanderbez Just to get this right: assuming we remove the `client.Context` arg here and use `clientCtx := client.GetClientContextFromCmd(cmd)`, we can still merge this PR before #6573, correct?

Reply:
If tests pass, yes 👍